src/mixins/amalgamation-mixin.ts

```diff
@@ -222,11 +253,12 @@ export default class AmalgamationMixin extends Vue {
           role: AmlRoles.AMALGAMATING,
           identifier: tingBusiness.businessInfo.identifier,
           name: tingBusiness.businessInfo.legalName,
-          email: tingBusiness.authInfo?.contacts[0].email || null,
+          email: tingBusiness.authInfo?.contacts[0].email,
           legalType: tingBusiness.businessInfo.legalType,
-          address: tingBusiness.addresses?.registeredOffice.mailingAddress || null,
```

Comment:
I don't remember why we added these fallbacks. I think we don't need them -- the property will be "falsy" either way.

On the other hand, if we do need the fallbacks, then they should be added to the [Amalgamating Businesses component](https://github.com/bcgov/business-create-ui/blob/50822182b2c6936ff5341f0aa488a99256f9cd2a/src/components/Amalgamation/AmalgamatingBusinesses.vue#L376) as well. LMK

Reply:
I think it was here Sev: https://github.com/bcgov/business-create-ui/pull/608#discussion_r1442107071

I'm OK with this.

Reply:
Thanks for finding that old review comment. I still think it's not needed. Resolving.